mobile/src/components/common/customscrollview/customscrollview.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC, ReactNode } from 'react';
+import { ScrollView } from 'react-native';
+
+type Props = {
+  children: ReactNode;
+};
+
+const CustomScrollView: FC<Props> = ({ children }) => {
+  return (
+    <ScrollView keyboardShouldPersistTaps="handled">{children}</ScrollView>
+  );
+};
```

Comment:
It should accept same props
```suggestion
const ScrollView: FC<Props> = (props) => {
  return (
    <UIScrollView keyboardShouldPersistTaps="handled" {...props} />
  );
};
```